main/src/ca/uwaterloo/flix/language/dbg/DocAst.scala

```diff
@@ -93,6 +95,8 @@ object DocAst {
 
     case class Match(d: Expr, branches: List[(Expr, Option[Expr], Expr)]) extends Atom
 
+    case class ExtMatch(d: Expr, branches: List[(Expr, Expr)]) extends Atom
```

Comment:
could you post what these two new pretty prints look like?

Reply:
I guess it still needs to be tidied up a bit